org.OpenT2T.Sample.SuperPopular.Thermostat/org.OpenT2T.Sample.SuperPopular.Thermostat.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE node PUBLIC "-//freedesktop//DTD D-BUS Object Introspection 1.0//EN" "http://standards.freedesktop.org/dbus/introspect-1.0.dtd"[]>
+<node>
+  <interface name="org.OpenT2T.Sample.SuperPopular.Thermostat">
+  
+    <!-- Turn on the HVAC -->
+    <method name="turnOn"/>
+    
+     <!-- Turn off the HVAC -->
+    <method name="turnOff"/>
+    
+    <!-- Get the current ambient temperature -->
+    <method name="getRoomTemperature">
+      <arg name="temp" type="d" direction="out" />
+    </method>
+
+     <!-- get the current cooling temperature (a.k.a. max setpoint) -->
+    <method name="getCoolingSetPoint">
+      <arg name="temp" type="d" direction="out" />
+    </method>
+    
+    <!-- get the current heating temperature (a.k.a. min setpoint) -->
+    <method name="getHeatingSetPoint">
+      <arg name="temp" type="d" direction="out" />
+    </method>
+  
+   <!-- set the desired cooling temperature (a.k.a. max setpoint) -->
+    <method name="setCoolingSetPoint">
+      <arg name="temp" type="d" direction="in" />
+    </method>
+    
+    <!-- set the desired heating temperature (a.k.a. min setpoint) -->
+    <method name="setHeatingSetPoint">
+      <arg name="temp" type="d" direction="in" />
+    </method>
+    
+   <!-- set the mode (e.g. Heating, Cooling, Auto)-->
+    <method name="setMode">
```

Comment:
I think an application would need a method to get the mode, and a way to list all the modes supported by a particular thermostat. But those things can be added on later I guess.